newsletter-december-2021.rst

```diff
@@ -0,0 +1,104 @@
+.. post:: December 14, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in December.
+
+Read the Docs newsletter - December 2021
+========================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We successfully deployed mitigation measures against spam,
+  and we are happy to report that the amount of abusive projects
+  has dramatically decreased.
+- We started tracking usage of different parts of our application
+  in a more systematic fashion which will enable us to make
+  better decisions going forward.
+
+New features
+------------
+
+- We revamped our support for :doc:`webhooks <readthedocs:build-notifications>`:
+  now they have a dedicated section in the project settings,
+  you can customize the payload using the Django template syntax,
```

Comment:
```suggestion
  you can customize the payload as yo wish
```

The only thing about the syntax is using `{{ }}`, I wouldn't call it django template syntax, it will be confusing that nothing else form the django's template syntax isn't supported.